tools/tasks/copyChangelog.js

```diff
@@ -0,0 +1,88 @@
+const fs = require('fs');
+const showdown = require('showdown');
+const colors = require('ansi-colors');
+const fancyLog = require('fancy-log');
+const OPTIONS = require('../options');
+
+/**
+ * Loads the changelog markdown into a string, and then returns it.
+ *
+ * @method loadChangelogMarkdown
+ * @returns {String} the raw markdown of the changelog
+ */
+function loadChangelogMarkdown() {
+    const sourceFile = OPTIONS.FILE.CHANGELOG_SOURCE;
+    const sourceMarkdown = fs.readFileSync(sourceFile, { encoding: 'utf8' });
+
+    return sourceMarkdown;
+}
+
+/**
+ * Strips out everything but the most recent changelog entry.
+ *
+ * @method getLatestEntry
+ * @param {String} markdown
+ * @returns {String} the data from the latest entry
+ */
+function getLatestEntry(markdown) {
+    const headerRegexp = /# [0-9]\.[0-9]+\.[0-9] \(.*\)/g;
```

Comment:
we should consider moving this to `globalconstants.js` `REGEX`. maybe...I am not sure.

Reply:
definitely